src/theme/index.tsx

```diff
@@ -1,4 +1,3 @@
-import { transparentize } from 'polished'
 import React from 'react'
```

Comment:
don't change this file, mod it as the other examples do

Reply:
Reverted changes.